src/dvc_task/app/filesystem.py

```diff
@@ -5,13 +5,16 @@
 
 from celery import Celery
 from kombu.message import Message
+from kombu.transport.filesystem import Channel
 from kombu.utils.encoding import bytes_to_str
 from kombu.utils.json import loads
 
 from ..utils import makedirs, remove, unc_path
 
 logger = logging.getLogger(__name__)
 
+Channel.QoS.restore_at_shutdown = False
```

Comment:
Can we add a comment with the reason/issue link for https://github.com/iterative/dvc-task/issues/61 to note why this is needed (also so we remember to remove it if/when the underlying ack issue gets fixed in celery)